go.mod

```diff
@@ -1,11 +1,13 @@
 module github.com/ssvlabs/eth2-key-manager
 
-go 1.18
+go 1.21.0
```

Comment:
can we specify 1.21 to not lock into a minor version, or it doesnt really matter?